src/main/java/com/revature/studyforce/flashcard/service/AnswerService.java

```diff
@@ -0,0 +1,124 @@
+package com.revature.studyforce.flashcard.service;
+
+import com.revature.studyforce.flashcard.dto.AnswerDTO;
+import com.revature.studyforce.flashcard.model.Answer;
+import com.revature.studyforce.flashcard.model.Flashcard;
+import com.revature.studyforce.flashcard.repository.AnswerRepository;
+import com.revature.studyforce.flashcard.repository.FlashcardRepository;
+import com.revature.studyforce.user.model.User;
+import com.revature.studyforce.user.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.util.Locale;
+import java.util.Optional;
+
+/**
+ * Services for the Answer repository {@link AnswerRepository}
+ * @author Edson Rodriguez
+ */
+@Service
+public class AnswerService  implements AbstractService{
```

Comment:
on most methods in this class, remember to {@link} to the methods your method is dependant on, mostly linking to the respective repository method